media/css/firefox/all/all-unified.scss

```diff
@@ -178,7 +183,31 @@ $image-path: '/media/protocol/img';
     }
 }
 
+.c-download-info {
+    border-bottom: 1px solid rgba(0, 0, 0, .2);
+    margin-bottom: $layout-sm;
+    width: 100%;
+}
+
+.c-download-info-caption {
+    @include field-label();
+    display: table-caption;
+}
+
+.c-download-info-label {
+    font-weight: normal;
+    // make first column as narrow as possible https://stackoverflow.com/a/11267268/462195
+    width: 1%;
+    white-space: nowrap;
+}
+
+.c-download-info-content {
+    font-weight: bold;
+}
+
 .c-download {
+    text-align: right;
```

Comment:
Might need a bidi mixin for RTL locales here?

Reply:
Updated.